src/styles/index.css

```diff
@@ -0,0 +1,118 @@
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
```

Comment:
The images are not being loaded from a specific folder. Use a relative path to load images from a specific folder.